Supertext.Base.Dal.SqlServer.Tests/Utils/SqlResultConverterTest.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Supertext.Base.Dal.SqlServer.Utils;
+
+namespace Supertext.Base.Dal.SqlServer.Tests.Utils;
+
+[TestClass]
+public class SqlResultConverterTest
+{
+    [TestMethod]
+    public void Level1DateToUtc()

```

Comment:
For test method names, we use the syntax of `MethodName_When_Then()`.
Here it would be like `InterpretUtcDates_DatesInUtcFormat_ConvertedInUtc`